sdk/RELEASE.md

```diff
@@ -11,6 +11,7 @@
 
 ## Documentation updates
 * Make full version dropdown visible on all KFP SDK docs versions [\#10577](https://github.com/kubeflow/pipelines/pull/10577)
+* Drop support for Python 3.7 since it has reached end-of-life(part1). [\#10750](https://github.com/kubeflow/pipelines/pull/10750)
```

Comment:
I think this should be considered as a breaking change. Other than that, the change looks good to me.